recipe/meta.yaml

```diff
@@ -1,54 +1,42 @@
 {% set name = "graphql-core" %}
-{% set version = "1.1" %}
-{% set bundle = "tar.gz" %}
+{% set version = "2.0" %}
 {% set hash_type = "sha256" %}
-{% set hash = "63bb8593aeeadb0a53e14207b910027fe51158d017927fad87326dac806185ee" %}
-{% set build = 1 %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4830699be53f9154273fa15726fc8b0c90bc22bbb8fc7c932586503b3cb9330e
 
 build:
   noarch: python
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
-  build:
+  host:
     - python
-    - setuptools
-
+    - pip
   run:
     - python
     - six >=1.10.0
-    - promise >=2.0
+    - promise >=2.1
+    - rx >=1.6.0
 
 test:
   imports:
     - graphql
-    - graphql.error
-    - graphql.execution
-    - graphql.execution.executors
-    - graphql.execution.experimental
-    - graphql.language
-    - graphql.pyutils
-    - graphql.type
-    - graphql.utils
-    - graphql.validation
-    - graphql.validation.rules
 
 about:
   home: https://github.com/graphql-python/graphql-core
-  license_file: {{ RECIPE_DIR }}/LICENSE
   license: MIT
   license_family: MIT
-  summary: 'GraphQL implementation for Python'
+  license_file: LICENSE
+  summary: 'GraphQL base implementation for Python'
+  doc_url: http://graphql.org/
```

Comment:
I wouldn't add this at this time: while the upstream links to it, you won't really be able to apply knowledge 1-to-1 from there to using this library...